codegen/src/test/resources/q4/Q4Aggregate1ProcessFunction.scala

```diff
@@ -0,0 +1,17 @@
+import org.hkust.RelationType.Payload
+import org.apache.flink.api.common.state.ValueStateDescriptor
+import org.apache.flink.api.common.typeinfo.{TypeHint, TypeInformation}
+import org.hkust.BasedProcessFunctions.AggregateProcessFunction
+class Q4Aggregate1ProcessFunction extends AggregateProcessFunction[Any, Integer]("Q4Aggregate1ProcessFunction", Array("ORDERKEY"), Array("O_ORDERPRIORITY"), aggregateName = "DIST_COUNT", deltaOutput = true) {
```

Comment:
Add nextKey : {O_ORDERPRIORITY} into the json file in AggregateProcessFunction 1, 

which will change this line to 

class Q4Aggregate1ProcessFunction extends AggregateProcessFunction[Any, Integer]("Q4Aggregate1ProcessFunction", Array("ORDERKEY"), Array("O_ORDERPRIORITY"), nextKey=Array("O_ORDERPRIORITY"), aggregateName = "DIST_COUNT", deltaOutput = true) {

Reply:
I thought there is no next_key anymore in aggregate process functions?